public/userInfo.php

```diff
@@ -244,10 +247,17 @@ function resize() {
             }
             echo "<a class='btn btn-link' href='/createmessage.php?t=$userPage'>Message</a>";
 
-            if (GetFriendship($userPage, $user) == UserRelationship::Following) {
-                echo "<span class='px-3'>Follows you</span>";
+            if ($areTheyFollowingMe) {
+                echo "<span class='hidden sm:block px-3'>Follows you</span>";
             }
+
             echo "</div>";
+
+            // This needs to be a sibling to the user relationship div to prevent spacing
+            // issues with the user avatar.
+            if ($areTheyFollowingMe) {
+                echo "<p class='sm:hidden'>Follows you</p>";
+            }
```

Comment:
Why does this need to be present twice? (line 251 + line 259)

Reply:
A bug in our Tailwind version was causing the class name `sm:justify-start` to not work properly, which was prohibiting this paragraph tag from fitting into a single flex container.

I have bumped our version to the latest minor release which fixes this issue, allowing me to use a single flex container and a single paragraph tag.